### DISCUSSION TITLE
intellij idea without chat function in business subscription

### DISCUSSION THREAD

**Discussion by wangliguang111** [community/community#109288] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Business subscription intellij idea does not have chat functionality. Our business account was purchased in November 2023 and as of now there is no chat function, which seriously affects our use. A business account isn't even as good as a personal account, some of the personal accounts that are added to waitlist already have chat functionality.

**Comment by davecheney** [comment, marked as answer]
@wangliguang111 currently intellij chat is a waitlisted feature, https://github.blog/changelog/2023-11-08-copilot-in-jetbrains-ides-chat-is-now-available-in-private-beta-new-welcome-guide/. This is a highly subscribed beta, we are adding new users as capacity permits.
